### PR TITLE
Add ChangeLog from preview2 to preview3

### PR DESCRIPTION
**Pull request by Qiao-Jin**

### CODE REVIEW

**Comment by superboyiii** [comment]
@erikzhang This is the last PR for neo-core Preview3, please have a look.

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+- Optimize the GetPrice() mechanism of SYSCALLs
+- Decouple NativeContract from InteropService
+- Subscribe to RelayResult messages in ConsensusService
+- Modify iterators to public
+- Move cosigners to transaction attributes
+- Make NEP6Wallet.path public
+- Unify unhandled exceptions
+- Remove FromJson
+- Unify akka logs with neo logs
+- Update neo-vm package
```
Comment (shargon) [inline CHANGELOG.md:41]: I think that we can avoid the update `neo-vm` we have multiple entries like this

Comment (Qiao-Jin) [reply CHANGELOG.md:41]: Have removed duplicate ones.

**Comment by superboyiii** [comment]
Looks good!

**Comment by superboyiii** [comment]
@erikzhang @shargon  Shall we merge?

**Comment by shargon** [comment]
@superboyiii Wait for other's review please

**Comment by superboyiii** [comment]
> I think there is a changelog in neo-node. We should use it.
> 
> https://github.com/neo-project/neo-node/blob/master/CHANGELOG.md

We plan to add changelog to each repository since if some repository's changelogs are added to neo-node is not appropriate. We'll update the other repositories soon.

**Pull request closed without merging**